wake-lock/wakelock-onactivechange.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Wake Lock 'onactivechange' Test</title>
+<link rel="author" title="Intel" href="http://www.intel.com">
+<link rel="help" href="https://w3c.github.io/wake-lock/">
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+
+<script>
+
+promise_test(t => {
+  let request, wakelock, eventWatcher;
+  return navigator.getWakeLock("screen")
+  .then(wakeLock => {
+    wakelock = wakeLock;
+    assert_false(wakelock.active, "the active is false before wake lock is acquired");
+    eventWatcher = new EventWatcher(t, wakelock, 'activechange');
```

Comment:
Which script introduces EventWatcher? Never seen that. Maybe add a comment?

Reply:
[testharness.js]( http://web-platform-tests.org/writing-tests/testharness-api.html)